### DISCUSSION TITLE
first of all thankyou very much for your very nice project, waiting for improvements...

### DISCUSSION THREAD

**Discussion by lucdeluc** [a1ive/grub2-filemanager#331] [General]
really interesting your project, for the moment i use your iso from Ventoy that is really extraordinary but is limited to the drive from it was booted,  maybe in future we can admire a sort of fusion of them?
Or your project is to be intended as an extenxion of ventoy?

**Comment by a1ive** [comment]
> but is limited to the drive from it was booted

This is the limitation of Ventoy. grubfm can be booted from anywhere and can read all disks.

**Reply by lucdeluc** [reply]
yes, yes absolutly agree regarding this.
your grubfm is really an improvement

**Comment by lucdeluc** [comment]
yes yes absolutly i agree with you this is a real useful improvement
thankyou very much
